src/views/PrivacyNotice.vue

```diff
@@ -0,0 +1,40 @@
+<template>
```

Comment:
Jeg tror ikke jeg vil ha denne siden live før vi faktisk har skrevet en skikkelig personvernserklæring. Her kan det hende vi burde hente inn hjelp fra noen som kan dette skikkelig.

Reply:
Både enig og uenig her. Er på en side lurt at personvernerklæringen er godt skrevet, helst med hjelp fra noen som kan det, men på en annen side er det litt ugreit å ha siden live i for lang tid uten noen som helst informasjon på dette feltet. Balansen mellom disse to er litt vanskelig. 

Men dette bør nok uansett være i et eget issue (jeg bare slang med fordi du nevnte det i issuet), da kan vi kan få _logg inn med mer kjærlighet_ live før man blir helt enige om innholdet i en personvernerklæring.

Reply:
Jeg åpnet issue #181, jeg.

Reply:
Lukker denne diskusjonen, videre diskusjon om dette tas i issue #181 eller PR #182.